resources/views/layouts/default/auth/register.blade.php

```diff
@@ -6,54 +6,74 @@
     <div class="col-sm-3"></div>
     <div class="col-sm-6">
 
-      {{ Form::open(['url' => '/register', 'class' => 'form-signin']) }}
-
+      <form method="post" action="{{ url('/register') }}" class="form-signin">
```

Comment:
Did you tested this @arthurpar06 ?

I have reports about this form returning back to the same page without registering users, trying to replicate it on local and dev environments.

Reply:
As discussed on discord it was a typo error in your theme right ?

Reply:
Exactly, fixed.